src/Sulu/Bundle/SecurityBundle/Tests/app/config/config.yml

```diff
@@ -1,76 +1,10 @@
 framework:
-    secret: secret
-    router: { resource: "%kernel.root_dir%/config/routing.yml" }
-    templating: { engines: ['twig'] }
-    test: ~
-    session:
-        storage_id: session.storage.mock_file
     profiler:
-        enabled: false
+        enabled: true
 
-fos_rest:
-    routing_loader:
-        default_format: json
+security:
+    access_control:
+        - { path: ^/security/reset, roles: IS_AUTHENTICATED_ANONYMOUSLY }
 
```

Comment:
cant't use an unauthenticated user. although i made i stated the the route should be publicly available